warehouse/upload.go

```diff
@@ -232,6 +241,31 @@ func (job *UploadJobT) generateUploadSchema(schemaHandle *SchemaHandleT) error {
 	return err
 }
 
+// excess columns are included in excluded schema
+func (job *UploadJobT) generateExcludedSchema() {
```

Comment:
rather than accessing the global state within the function, can we pass the limit as the param to this function ?

`generateExcludedSchema(limit int)` or something. Wdyt ?

Reply:
done.